src/main/resources/application.properties

```diff
@@ -9,5 +9,5 @@ spring.jpa.open-in-view=false
 
 spring.jackson.deserialization.fail-on-unknown-properties=true
 
-jwt.expiration=300000
+jwt.expiration=3000000000
```

Comment:
I think you forgot to revert it back to 5 minutes.
For testing I suggest to change SecurityConfig to permit all requests, it's easier for me.

Reply:
i will set a normal expiration once the project is finished))

Reply:
it's not related to this hw and pr.